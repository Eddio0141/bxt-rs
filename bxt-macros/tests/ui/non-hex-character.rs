use bxt_macros::pattern;

fn main() {
    pattern!(00 0v);
}
