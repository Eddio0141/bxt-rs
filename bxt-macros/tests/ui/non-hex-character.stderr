error: token must be a hex number
 --> $DIR/non-hex-character.rs:4:17
  |
4 |     pattern!(00 0v);
  |                 ^^
